src/gui/gt_mdilauncher.cpp

```diff
@@ -411,6 +411,22 @@ GtMdiLauncher::isPrintable(QWidget* subWindow) const
     return false;
 }
 
+std::vector<GtQmlAction*>
```

Comment:
same here regarding constness
```suggestion
std::vector<const GtQmlAction*>
```